### PR TITLE
Add WG members to README

### PR DESCRIPTION
**Pull request by tobie**
/cc @brianwarner, @CrystalOnScript, @Gregable, @jorydotcom, @nainar, @rsimha, @sebastianbenz.

@mrjoro will add you to the relevant GH team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,15 @@ The tentative date for graduation is December 10, 2019.
...
-- [@tobie](https://github.com/nainar) - Tobie Langel (Facilitator)
+- [@brianwarner](https://github.com/brianwarner) - Brian Warner
+- [@CrystalOnScript](https://github.com/CrystalOnScript) - Crystal Lambert
+- [@Gregable](https://github.com/Gregable) - Greg Grothaus
 - [@jrozier](https://github.com/jrozier) - Joey Rozier
+- [@jorydotcom](https://github.com/jorydotcom) - Jory Burson
+- [@nainar](https://github.com/nainar) - Naina Raisinghani 
+- [@rsimha](https://github.com/rsimha) - Rhagu Simha
+- [@sebastianbenz](https://github.com/sebastianbenz) - Sebastian Benz
+- [@tobie](https://github.com/nainar) - Tobie Langel (Facilitator)
```
Comment (mrjoro) [inline README.md]:
```suggestion
- [@tobie](https://github.com/tobie) - Tobie Langel (Facilitator)
```

**Review by mrjoro — APPROVED** [review]
LGTM, though we should fix your URL. :)

**Comment by mrjoro** [comment]
It looks like GitHub let me commit that URL fix myself!

File: README.md [hunk 2]
```diff
@@ -12,8 +12,15 @@ The tentative date for graduation is December 10, 2019.
...
 
 # Members
-- [@tobie](https://github.com/nainar) - Tobie Langel (Facilitator)
+- [@brianwarner](https://github.com/brianwarner) - Brian Warner
+- [@CrystalOnScript](https://github.com/CrystalOnScript) - Crystal Lambert
+- [@Gregable](https://github.com/Gregable) - Greg Grothaus
 - [@jrozier](https://github.com/jrozier) - Joey Rozier
+- [@jorydotcom](https://github.com/jorydotcom) - Jory Burson
+- [@nainar](https://github.com/nainar) - Naina Raisinghani 
+- [@rsimha](https://github.com/rsimha) - Rhagu Simha
```
Comment (rsimha) [inline README.md:21]: `s/Rhagu/Raghu/`

**Pull request merged**